src/components/Event.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Calendar from '../assets/Calendar.png';
+
+interface EventProps {
+  eventName: string;
+  eventDate: string; // Assuming the date is provided as a string in '1/2/2024' format
+  eventDescription: string; // Description is not pictured in reference images
+}
+
+const Event: React.FC<EventProps> = ({ eventName, eventDate}) => {
+  // Format the date for display
+  const formattedDate = new Date(eventDate).toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className = "event-container">
+        <img src={Calendar} width={30} height={30} alt="Description of the image" style={{flexShrink: 0}} />
```

Comment:
why are you setting style here?

Reply:
I was copying the CSS from figma - I suppose the idea is to ensure that the image doesn't shrink? it may not be neccesary